_wiki/FAQ/Windows.md

```diff
@@ -5,186 +5,157 @@ hide_from_auto_list: true
- - Delete or move the `settings.json` file from inside this folder.
- - If this persists after also remove the `Plugins` folder.
+Check if `OpenTabletDriver.Daemon` and `OpenTabletDriver.UX.Wpf` is in the same folder. This should be the case if the [Installation]({% link _wiki/Install/Windows.md %}) instructions were followed correctly.
 
-If you have a `userdata` folder next to the binaries of OpenTabletDriver, you will need to remove the files above from that folder instead.
+If both files are in the same folder, the daemon may have crashed during launch.
 
-If you still can't get it working after the solutions above please join our [Discord]({{ site.discord_invite_url }}).
-It might be a completely different issue!
+- Press <kbd>Win</kbd> + <kbd>R</kbd> to open the Run dialog.
+- Type `%localappdata%\OpenTabletDriver` then press <kbd>Enter</kbd>.\\
+- Delete or rename `settings.json` to `settings.json.bak`.
+- If problem persists, try also renaming `Plugins` folder to `Plugins.bak`.
+
+> Note: If you converted OpenTabletDriver to a portable install, go inside `userdata` folder instead.
 
 ---
 
-### How do I make OTD start automatically when I boot my PC? {#startup}
+### How do I make OpenTabletDriver to start automatically when I turn on my computer? {#startup}
+
+The following is just one of the multiple ways that you may launch OTD at startup.
 
-Starting OpenTabletDriver at login can be performed quite easily. This can be done for nearly any
-program, although there may be better ways to do it for those programs.
+- Right click `OpenTabletDriver.UX.Wpf.exe` and select `Create shortcut`.
+- Press <kbd>Win</kbd> + <kbd>R</kbd> to open the Run dialog.
+- Type `shell:startup` then press <kbd>Enter</kbd>.
+- Move the shortcut you created earlier to the folder that opened.
 
- - Right click `OpenTabletDriver.UX.Wpf.exe` and create a shortcut.
- - Press <kbd>Win</kbd> + <kbd>R</kbd> and type in `shell:startup`
- - Move the shortcut into the folder that opened.
+To start OpenTabletDriver minimised, continue below:
 
-Starting OpenTabletDriver minimised can also be done by changing the properties of a shortcut.
-To do this, right click the OpenTabletDriver shortcut, go to properties > run > minimised.
+- Right click the shortcut and select `Properties`.
+- In the `Run` dropdown, select `Minimized`.
 
 ---
 
-### Plugin Manager won't open/I get a warning on startup? {#administrator}
+### I get a warning about not running it as Administrator on startup? {#administrator}
 
-When Starting OpenTabletDriver you may get an error warning you about Administrator Permissions.
-OpenTabletDriver should NEVER be run as Administrator or with Administrator permissions.
-This causes the plugin manager, FAQ page, and Tablet debugger to not work, and can interfere with several other features.
+OpenTabletDriver should *NEVER* be run as Administrator, and there is never a need to run as Administrator.
```

Comment:
there is a need if the application you're running (for some reason) is running as admin (e.g. task manager?), maybe throw in a mention of VMulti here?